pkg/bundle/read.go

```diff
@@ -117,6 +119,20 @@ func read(ctx context.Context, name, baseDir string, bundleSpecReader io.Reader,
 	if err := yaml.Unmarshal(bytes, bundle); err != nil {
 		return nil, err
 	}
+
+	var scans []*fleet.ImageScan
+	for _, scan := range bundle.ImageScans {
+		if scan.Image == "" {
+			continue
+		}
+		scans = append(scans, &fleet.ImageScan{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: fmt.Sprintf("%s-%s", name, name2.Hex(scan.Image, 5)),
+			},
+			Spec: scan,
+		})
+	}
```

Comment:
Will `scans` always have at least one scan? Or, is it possible that scans could remain unitialized?

Reply:
It could be empty though.

Reply:
Good point